scipy/stats/stats.py

```diff
@@ -7469,6 +7470,254 @@ def brunnermunzel(x, y, alternative="two-sided", distribution="t",
     return BrunnerMunzelResult(wbfn, p)
 
 
+def _confint_lowerbound(n, quantile, confidence):
+    r"""
+    Compute the lower bound for a one-sided confidence interval
+    for a given
+    - quantile (0<`quantile`<1)
+    - confidence level (0<`confidence`<1)
+    - number of samples `n`.
+
+    Returns the largest index of the sample being a valid lower bound,
+    or `None` if there are not enough samples to derive one.
+
+    Used by the public function confint_quantile().
+
+    .. versionadded:: 1.6.0
```

Comment:
```suggestion
    .. versionadded:: 1.7.0
```
We forgot to update this in the HiGHS PR. Easily forgotten, so I thought I'd point it out.